stories/DropdownMenu.stories.tsx

```diff
@@ -20,25 +20,25 @@ const Default: React.FC = () => (
       </DropdownMenuItem>
       <DropdownMenuDivider />
       <DropdownMenuItem>
-        <Icon iconName="minus-round" className="leftIcon" />
+        <Icon iconName="minus-round"/>
         Cut
       </DropdownMenuItem>
-      <DropdownMenuItem className="withRightIcon">
-        <Icon iconName="copy" className="leftIcon" />
+      <DropdownMenuItem>
+        <Icon iconName="copy"/>
         Copy
-        <Icon iconName="check" className="rightIcon" />
+        <i className="tk-dropdown-menu--selected" />
```

Comment:
It'd makes more sense to have this class attached to the whole item, than just the icon

Reply:
Actually I am not sure to understand why we don't use the <Icon component, is it a specific

Reply:
Yes, it could be a nice improvement to have a prop selected?= true